packages/slate-editor/package-lock.json

```diff
@@ -1346,44 +1337,6 @@
                 }
             }
         },
-        "@prezly/slate-commons": {
-            "version": "0.1.8",
-            "resolved": "https://registry.npmjs.org/@prezly/slate-commons/-/slate-commons-0.1.8.tgz",
-            "integrity": "sha512-G7Oi/lvWLRx8WWgd41ho5sL2JzkmqqxZB4GKjxLqHP5PxPyDD+6h+CcCkvOqTwwULMHLbyV/oRdAKDc1lYGbGw==",
-            "dev": true,
-            "requires": {
-                "uuid": "^8.3.2"
-            }
-        },
-        "@prezly/slate-hyperscript": {
-            "version": "0.1.8",
-            "resolved": "https://registry.npmjs.org/@prezly/slate-hyperscript/-/slate-hyperscript-0.1.8.tgz",
-            "integrity": "sha512-VM53+8X6yERHonHd6vPH1v8t43HhkzZX4C7IyC9FcHGrLgEooIfHCFB/D/EC8xTx1RHrJZyn+GnsSR1tytqTYg==",
-            "dev": true,
-            "requires": {
-                "is-plain-object": "^5.0.0"
-            }
-        },
-        "@prezly/slate-lists": {
```

Comment:
@kamilmielnik for some reason Lerna keeps removing these dependencies. The only way I've found to keep them is to run `npm install` in the sub-packages dirs manually (in #32)  :shrug: